homeassistant/components/asuswrt/config_flow.py

```diff
@@ -132,9 +132,6 @@ async def _async_check_connection(self, user_input):
 
     async def async_step_user(self, user_input=None):
         """Handle a flow initiated by the user."""
-        if self._async_current_entries():
```

Comment:
We probably need to add a guard now for setting up the same host more than once.